spec/models/contact_spec.rb

```diff
@@ -1121,13 +1121,13 @@ def contacts_with_value_at(value,time)
         )
         HistoryEntry.create(attribute: 'level',
                             old_value: Contact::VALID_LEVELS['sádhaka'],
-                            changed_at: '2012-11-26 18:58:21 UTC'.to_time,
+                            changed_at: '2012-11-27 18:58:21 UTC'.to_time,
                             historiable_type: 'Contact',
                             historiable_id: @contact._id
         )
         HistoryEntry.create(attribute: 'level',
                             old_value: Contact::VALID_LEVELS['aspirante'],
-                            changed_at: '2012-11-26 23:41:16 UTC'.to_time,
+                            changed_at: '2012-11-28 23:41:16 UTC'.to_time,
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `to_time` on Date objects, because they know nothing about the time zone in use.